plugins/snapshots/devmapper/dmsetup/dmsetup.go

```diff
@@ -415,7 +415,8 @@ func parseDmsetupError(output string) string {
 
 	line := lines[0]
 	// Handle output like "Device /dev/mapper/snapshotter-suite-pool-snap-1 not found"
-	if strings.HasSuffix(line, "not found") {
+	// Handle output like "Device does not exist.\nCommand failed.\n"
+	if strings.HasSuffix(line, "not found") || strings.HasPrefix(line, "Device does not exist") {
```

Comment:
do we know when the error shows `"not found"` and when `Device does not exist`?

Reply:
I find some kernel log 
```
2025-06-09T04:08:40.5195139Z [Mon Jun  9 04:08:14 2025] device-mapper: thin: Creation of new thinly-provisioned device with id 1 failed.
2025-06-09T04:08:40.5195614Z [Mon Jun  9 04:08:14 2025] loop4: detected capacity change from 0 to 16384
2025-06-09T04:08:40.5196007Z [Mon Jun  9 04:08:14 2025] device-mapper: thin: Deletion of thin device 2 failed.
2025-06-09T04:08:40.5196629Z [Mon Jun  9 04:08:14 2025] device-mapper: thin: 252:0: reached low water mark for data device: sending event.
```
 @djdongjin

Reply:
dmsetup info failed but I can't find reason.
```
dmsetup info --columns --noheadings -o name,blkdevname,attr,major,minor,open,segments,events --separator   containerd-snapshotter-suite-pool-108175915-snap-1
Device does not exist.\nCommand failed.\n" 
```